### PR TITLE
Await the outcome of the image build tasks

### PR DESCRIPTION
**Pull request by k-wall**
Collects the task-ids of the underlying brew tasks and await them all.  Image builds still dispatched in parallel.

### CODE REVIEW

**Comment by k-wall** [comment]
@vbusch I re-used `CEKIT_OUTPUT_FILE`.  It seemed like this file was never actually written.  I now write it using `tee` command.

**Comment by vbusch** [comment]
> @vbusch I re-used CEKIT_OUTPUT_FILE. It seemed like this file was never actually written. I now write it using `tee` command.

The nightly build uses CEKIT_OUTPUT_FILE.  
It uses tee to write to it: https://gitlab.cee.redhat.com/msgqe/amq-ci/blob/master/amq-online/src/pipelines/downstream/downstream-pnc.groovy#L109
And you can see it reading from it in the console output: https://maas-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/amq-online-downstream/job/nightly/job/amq-online-pnc-ci/693/consoleFull

File: Makefile [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
-	iot-http-adapter \
-	iot-lorawan-adapter \
-	iot-mqtt-adapter \
-	iot-proxy-configurator \
-	iot-sigfox-adapter \
-	iot-tenant-service \
-
-
-all: $(SUBDIRS)
+	#agent \
```
Comment (vbusch) [inline Makefile]: These modules shouldn't be commented out

**Review by vbusch — CHANGES_REQUESTED** [review]

**Comment by k-wall** [comment]
> > @vbusch I re-used CEKIT_OUTPUT_FILE. It seemed like this file was never actually written. I now write it using `tee` command.
> 
> The nightly build uses CEKIT_OUTPUT_FILE.
> It uses tee to write to it: https://gitlab.cee.redhat.com/msgqe/amq-ci/blob/master/amq-online/src/pipelines/downstream/downstream-pnc.groovy#L109
> And you can see it reading from it in the console output: https://maas-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/amq-online-downstream/job/nightly/job/amq-online-pnc-ci/693/consoleFull

Ok - I see now.  It is surprising because we have it declared in two places. I think I can refactor and make simpler.

File: Makefile (hunk 1 above)
Comment (k-wall) [reply Makefile]: ah.. sorry that was my short cut testing!  I'll fix.

**Comment by k-wall** [comment]
@vbusch I took an alternative approach, more along the lines you took.  I gave Makefile.common full control of the cekit.log file which then gets used by the the two later targets.  It now waits for the brew task individually rather than as a group, but the image builds will still be in parallel on the backend.

**Review by lulf — APPROVED** [review]

**Review by vbusch — APPROVED** [review]
LGTM

**Pull request merged**